index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute Page</title>
+</head>
+<body>
+    <div id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
```

Comment:
Use semantic tags instead of div tags.

Reply:
I think it will be better to write in polite suggestion instead of order. Sure it will be better to include an explanation for your comment. Please,  use "why", for each requested change 🤓